[DRAFT] TIRRC form

coreyar opened the pull request:
The new is dived up into  7 parts plus a signature section.

The form is pinned in the power-of-attorney channel on the Code for Nashville slack.
https://nashvillebrigade.slack.com/files/U2AK5BYNP/FH1DHFGGK/poa_sm_-_tirrc.docx

A logical way to break up the work is to divide by work on the react app, work on the PDF forms, and work on the translation files.


| Section                                             | Translate Text | Build Components | Format Form |
|-----------------------------------|----------------|------------------|-------------|
| 1. Child Information.            | []                       | [x]                         | [x]                  |
| 2. Parent contact information         | []                       | [x]                         | [x]                  |
| 3. Caregiver's information  | []                       | [x]                         | [x]                  |
| 3. Parental status                         | [ ]                       | [x]                         | [x]                  |
| 4. Temporary caregiving authority | [ ]                       | [x]                         | [ ]                  |
| 5. Rights                                           | [ ]                      | [ ]                          | [ ]                 |
| 6. Does not provide legal custody | [ ]                       | [ ]                         | [ ]                  |
| 7. Termination                                  | [ ]                       | [ ]                         | [ ]                  |

coreyar committed:
Section one/ multiple children one form

coreyar committed:
Caregiver step

coreyar committed:
Upgrade react

coreyar committed:
Disclaimer manages its own visibility

coreyar committed:
Add initial and successor caregiver w/ cleanup

coreyar committed:
flow

coreyar committed:
lint

coreyar committed:
Stub out steps

coreyar committed:
Step 5 formatting (#115)

* Step 5 formatting

* Empty string

combinatorist commented:
Oh - wow - we have a long way to go (thanks for making this grid of needs). Do want help on this? I can put some time in and recruit others.

combinatorist commented:
FYI: The form is pinned in the power-of-attorney channel on the Code for Nashville slack.
https://nashvillebrigade.slack.com/files/U2AK5BYNP/FH1DHFGGK/poa_sm_-_tirrc.docx

coreyar committed:
use html2pdf to make coding the pdf easier (#122)

* spanish translation, Add privacy notice to disclaimer page

* update yarn.lock

* use html2pdf to make coding the pdf easier

* Set defaults for development (#121)

* set defaults to dev

* lint

* explain how to set env variable via .env file in README

* only set mocked "child" name based on env variable

* update env per naming conventions

* Update README.md with doc link

* remove pdfMake

* Update Flow / minor fixes (#116)

* Handle multiple lines of text

* lint & flow

* remove comma

* Update type, upgrade flow

* seal objects

* revert changes

* automatic yarn lint fixes to screens/poa-form

* fix typo in README

* fix unnecessary escape in phoneNumber regex (for linting)

* drop extra blank line (for lint)

* fix linting

* lint

* Fix tests, improve flow

coreyar committed:
First section work

coreyar committed:
caregivers

coreyar committed:
flow

coreyar committed:
finish caregivers and start and custody

coreyar committed:
setp 3 form

coreyar committed:
Step five

mhspradlin committed:
Merge remote-tracking branch 'origin/master' into issue/110-tirrc-form

coreyar committed:
Merge pull request #130 from mhspradlin/issue/110-tirrc-form

Merge master into issue/110-tirrc-form

mhspradlin committed:
Updating to Grommet 2.0, refactoring steps into own files (#131)

* Updating to Grommet 2.0, refactoring steps into own files

* Separating theme into theme file

* Separating theme into theme file

* Adding Modal for error

* Adding final two steps of form

coreyar committed:
lint all files (#134)

coreyar committed:
Remove extra deps (#135)

* remove extra deps

* lint all files (#134)

* remove extra deps

coreyar committed:
Fix tests; minor updates (#138)

coreyar committed:
Minor form styles (#139)

* Update form input styles

* cleanup

ntbrewer committed:
Additions to TIRRC form (#141)

Added all parts of the form and formatted somewhat. the 'checkHere' parts may have a better CSS solution than my implementation and boxes are not yet filled appropriately. PDF strings should be correct and matched to TIRRC form.
